edx/edx-ora2
# Add missing expand/collapse indicators to assessment steps

**TL;DR -** Add missing expand/collapse arrows to assessment steps to provide better visual indication that these can be expanded/collapsed.

JIRA: Related to [EDUCATOR-5374](https://openedx.atlassian.net/browse/EDUCATOR-5374)

**What changed?**

- Add missing arrow to `Learn to Assess Responses` step when in-progress
- Add missing arrow to `Top Responses` step when it can be expanded
- Updated placeholder (loading) step titles to match loaded values

**Developer Checklist**


**Testing Instructions**

1. Create an ORA with at least `Learn to Assess Responses` step and `Show top responses` enabled
2. Submit a response
3. For each step in the process, verify that the `Learn to Assess Responses` step shows an expand/collapse arrow when appropriate (when clicking is able to expand/collapse)
4. For each step in the process, verify that the `Top Responses` section shows an expand/collapse arrow when appropriate (when clicking is able to expand/collapse)

**Before**
<img width="1112" alt="Screen Shot 2020-12-11 at 18 42 50" src="https://user-images.githubusercontent.com/12944465/101964829-b0cefb80-3be0-11eb-809a-62a028a7a855.png">

**After**
<img width="1112" alt="Screen Shot 2020-12-11 at 17 49 44" src="https://user-images.githubusercontent.com/12944465/101964682-2d150f00-3be0-11eb-8b23-08c0d223a88a.png">

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FIY: @edx/masters-devs-gta
